docs/data-sources/waf_policies.md

```diff
@@ -0,0 +1,86 @@
+---
+subcategory: "Web Application Firewall (WAF)"
+---
+
+# huaweicloud_waf_policies
+
+Use this data source to get a list of WAF policies.
+
+## Example Usage
+
+```hcl
+variable "policy_name" {}
+data "huaweicloud_waf_policies" "policies" {
+  name = var.policy_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) The region in which to obtain the WAF policies. If omitted, the provider-level region
+  will be used.
+
+* `name` - (Optional, String) Policy name used for matching. The value is case sensitive and supports fuzzy matching.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `ids` - A list of WAF Policy IDs.
```

Comment:
is this attribute is necessary, as we can get the ids from `policies.*.id`.

Reply:
removed the `ids`